src/inc/DirectoryObject.hpp

```diff
@@ -56,9 +56,18 @@ namespace MSIX {
 
         // IDirectoryObject
         ComPtr<IStream> OpenFile(const std::string& fileName, MSIX::FileStream::Mode mode) override;
-        std::multimap<std::uint64_t, std::string> GetFilesByLastModDate() override;
+        std::multimap<std::uint64_t, std::string> GetPayloadFilesByLastModDate() override;
 
     protected:
+        bool IsFootPrintFile(std::string normalized)

```

Comment:
>IsFootPrintFile [](start = 13, length = 15)

Not sure why this is here, but it doesn't seem like the responsibility of the directory object to filter package specific files.

Reply:
moved to packagewriter

---
In reply to: [278644877](https://github.com/Microsoft/msix-packaging/pull/95#discussion_r278644877) [](ancestors = 278644877)